conduit/plugins/importers/algod/algod_importer_test.go

```diff
@@ -289,30 +299,47 @@ netaddr: %s
 func TestGetBlockSuccess(t *testing.T) {
 	tests := []struct {
 		name        string
+		mode        string
 		algodServer *httptest.Server
 	}{
-		{"", NewAlgodServer(GenesisResponder,
-			BlockResponder,
-			BlockAfterResponder,
-			MakeSyncRoundResponder(http.StatusOK))},
-		{"archival", NewAlgodServer(GenesisResponder,
-			BlockResponder,
-			BlockAfterResponder,
-			MakeSyncRoundResponder(http.StatusOK))},
-		{"follower", NewAlgodServer(GenesisResponder,
-			BlockResponder,
-			BlockAfterResponder, LedgerStateDeltaResponder, MakeSyncRoundResponder(http.StatusOK))},
+		{
+			name: "default",
+			mode: "",
+			algodServer: NewAlgodServer(GenesisResponder,
+				BlockResponder,
+				BlockAfterResponder,
+				MakeSyncRoundResponder(http.StatusNotFound))},
+		{
+			name: "archival",
+			mode: "archival",
+			algodServer: NewAlgodServer(GenesisResponder,
+				BlockResponder,
+				BlockAfterResponder,
+				MakeSyncRoundResponder(http.StatusNotFound))},
```

Comment:
Most of these `http.StatusOK` to `http.StatusNotFound` changes led to tests that fail without the bugfix.